tests/foreman/rhai/test_rhai.py

```diff
@@ -35,15 +37,15 @@ def setUpClass(cls):  # noqa
         # Walk through the list of subscriptions.
         # Find the "Red Hat Employee Subscription" and attach it to the
         # recently-created activation key.
-        for subscription in org.subscriptions():
-            if subscription['product_name'] == DEFAULT_SUBSCRIPTION_NAME:
+        for subs in sub.search():
+            if subs.read_json()['product_name'] == DEFAULT_SUBSCRIPTION_NAME:
```

Comment:
FYI, you can do something like this:

```python
for result in sub.search_json()['results']:
    if result['product_name'] == DEFAULT_SUBSCRIPTION_NAME:
```

But don't consider this comment a blocker. I'm just letting you know of an alternate way of doing things that doesn't require calling `read_json()` on each result.